pkg/instrumentors/bpf/google/golang/org/grpc/probe.go

```diff
@@ -36,7 +36,7 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpfel -cc clang -cflags $CFLAGS bpf ./bpf/probe.bpf.c
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target amd64,arm64 -cc clang -cflags $CFLAGS bpf ./bpf/probe.bpf.c
```

Comment:
Why generate and embed eBPF in both architectures? One of them will never be used and will just increase the binary size. In Keyval's repo we passed `-target $TARGET` to `go:generate` and propagated it when compiling:
See https://github.com/keyval-dev/opentelemetry-go-instrumentation/blob/e237a41a45429e526559b376e499224d6f98c3dc/Dockerfile#L8

Reply:
I did originally do it the way keyval does it but felt it make the makefile and dockerfile a little messy by having to handle the TARGET. I didn't think about the overall image size and agree it would be better to be smaller / not have unneccary eBPF code. I'll switch back to passing in a target arch.